src/store/modules/search/actions.js

```diff
@@ -15,7 +17,19 @@ export const clearSearchedResults = async ({ commit, dispatch, getters }) => {
 }
 
 export const updateSearchedValue = async ({ commit, dispatch, state }, value) => {
-  value = String(value).replace(/[\W_]+/g, '')
+  if (value.match(/.rsk/)) {
+    try {
+      const address = await getAddr(value)
+      store.commit('SET_DOMAIN', { domain: value, address })
+
+      value = address
+    } catch (error) {
+      // console.error(error.message, value)
```

Comment:
This will avoid running `value = String…` to the value that made RNS fail. I guess we would like to continue the “old flow” if it fails

Reply:
It executes addr function only when the user provides a valid .rsk domain. Do we need to continue searching blocks, addresses, transactions with this search string (<domain>.rsk)?